openedx/features/course_experience/utils.py

```diff
@@ -4,22 +4,12 @@
 
 
 from completion.models import BlockCompletion
-from django.utils.translation import ugettext as _
```

Comment:
This file is the one that feeds the course_outline template context. And while there, my editor noticed all these unused imports. So I cleaned them up and reduced the pylint threshold a bit (see below). Not related, but I figured why not.